client_cli/src/main.rs

```diff
@@ -158,19 +168,34 @@ fn main() -> Result<()> {
                 .as_ref(),
         )?
     };
+
     let Configuration(config) = config;
-    println!(
-        "User: {}@{}",
-        config.account_id.name, config.account_id.domain_id
-    );
-    #[cfg(debug_assertions)]
-    eprintln!(
-        "{}",
-        &json5::to_string(&config).wrap_err("Failed to serialize configuration.")?
-    );
-    #[cfg(not(debug_assertions))]
-    eprintln!("This is a release build, debug information omitted from messages");
-    subcommand.run(&config)?;
+
+    if verbose {
+        eprintln!(
+            "User: {}@{}",
+            config.account_id.name, config.account_id.domain_id
+        );
+        #[cfg(debug_assertions)]
+        eprintln!(
+            "{}",
+            &json5::to_string(&config).wrap_err("Failed to serialize configuration.")?
+        );
```

Comment:
This is quaint. JSON5 is useful for accepting more input from the user (e.g. comments) without causing a deserialisation error. I think you could just use plain standard `json` here without losing anything major.

Reply:
It was probably done to avoid introducing another explicit dependency. Since we're unlikely to somehow not transitively depend on serde_json anyway, this can be changed as well.